### ISSUE TITLE
JDWP command failed for jni

### ISSUE THREAD

**Issue by hyx0408** [adelphes/android-dev-ext#136]
```
Prepared: Lcom/android/gles3jni/GLES3JNIView$Renderer;
Prepared: Lcom/android/gles3jni/GLES3JNIView$1;
Prepared: Lcom/android/gles3jni/GLES3JNILib;
JDWP command failed 'Linetable:GLES3JNILib,init'. Error 511
JDWP command failed 'Linetable:GLES3JNILib,resize'. Error 511
JDWP command failed 'Linetable:GLES3JNILib,step'. Error 511
```
Did this affect lldb to load the native jni library? After "System.loadlibrary("libmyjni")" run, there is no module named libmyjni.so in my lldb target. Do you know how to make lldb work with jni so?

**Comment by adelphes** [comment]
The debugger extension only works for pure Java and Kotlin code - the JDWP commands won't work for JNI libraries or native code.

Once the native library is loaded, you should be able to use lldb to debug the native code, but I think it requires the lldb binary to be copied the device, manually launched and then attached to as a remote process.

**Comment by hyx0408** [comment]
> The Android debugger extension only works for pure Java and Kotlin code - the JDWP commands won't work for JNI libraries or native code.
> 
> Once the native library is loaded, you should be able to use lldb to debug the native code, but I think it requires the lldb binary to be copied the device, manually launched and then attached to as a remote process.

Obviously, you are right. But now I'm suffering from two other problems. 
The first one is the JNI library is zipped into base.apk, so that it's a not normal library which lldb failed to load. 
I work around the problem by pushing the JNI library into /data/app/xx/xxx/lib/arm64 which is another path for finding a library for Android.
The second one is if I attach to the process and then the library loaded, lldb will not load it. However, in the local linux enviroment, after dlopen, the library will be loaded.
